api/src/main/data/bt_language_names.xml

```diff
@@ -414,6 +458,61 @@ QUESTIONS?
 <symbol>MALAYALAM</symbol>
 </language>
 <language>
+<id>107</id>
+<name>Marathi</name>
+<doc>Marathi</doc>
+<iso639-3>mar</iso639-3>
+<iso639-1>mr</iso639-1>
+<script>Deva</script>
+<symbol>MARATHI</symbol>
+</language>
+<language>
+<id>103</id>
+<name>Mongolian</name>
+<doc>Mongolian</doc>
+<iso639-3>mon</iso639-3>
+<iso639-1>mn</iso639-1>
+<script>Mong</script>
```

Comment:
In Mongolia, it is written in Cyrillic. In China, it is written in Mongolian. Which script should we specify for the macrolanguage?

Reply:
I believe we used Zyyy for Serbian due to it being in 2 scripts.  So I used that for the Mongolian macrolanguage script.